Userland/Libraries/LibJS/Runtime/MathObject.cpp

```diff
@@ -13,6 +13,18 @@
 #include <LibJS/Runtime/MathObject.h>
 #include <math.h>
 
+// These definitions are missing under cygwin
```

Comment:
You get these with a
```cpp
#define _USE_MATH_DEFINES
```
On windows
You can add this define as a cmake flag to avoid this issue in other places

Reply:
Yes, that's true for microsoft's compiler. We're currently using gcc which needs extensions to be turned on for it to work on windows. That would however break every class using `typeof` as a member name.

Reply:
we don't use `typeof `as a member name anywhere, its a restricted word afaik

Reply:
Inside of LibJS’s Value.h